apps/home/templates/authenticated-home.html

```diff
@@ -38,10 +38,7 @@
           <div class="ds-c-alert__body">
             <h3 class="ds-c-alert__heading">New!</h3>
               <p class="ds-c-alert__text">
-                <a href="https://www.npmjs.com/package/cms-bluebutton-sdk">Node</a>
-                and
-                <a href="https://pypi.org/project/cms-bluebutton-sdk/">Python</a>
-                SDKs are now available to help streamline integration with the Blue Button 2.0 API.
+                  <a href="https://bluebutton.cms.gov/developers/#authorization-screen-language-selection">New Spanish language parameter</a> is now available. Read more here about how to display the authorization screen in Spanish.
```

Comment:
Small question about the link here, did Product specifically want the link on `New Spanish language parameter`, or would it be better on `here` like this:
```suggestion
                  New Spanish language parameter is now available. Read more <a href="https://bluebutton.cms.gov/developers/#authorization-screen-language-selection">here</a> about how to display the authorization screen in Spanish.
```

Reply:
Originally, the link was requested on the first part of the new banner, but after rewording, that may not be the case. I'll double check and probably move to this.

Reply:
I fixed this!